builtin/game/register.lua

```diff
@@ -398,7 +398,9 @@ local function make_registration_wrap(reg_fn_name, clear_fn_name)
 
 	local orig_clear_fn = core[clear_fn_name]
 	core[clear_fn_name] = function()
-		list = {}
+		for k in pairs(list) do
+		 list[k]=nil
```

Comment:
> Indentation is one **tab**.  Also, use spaces around binary operators.  
> See http://dev.minetest.net/Lua_code_style_guidelines.

Thanks for pointing me to the right docs. You may as well ignore the patch 
(it is 3 lines after all) and take this pull as bugreport. But I thought it 
would be better to fix the problem even poorly rather than just talk about 
it.

Btw: Your signature  should be separated by line with **two** dashes and a space.

-- 
S pozdravom 4096R/C45F4253 Tomáš Brada (Born 1997-03-22 in Martin, Slovakia)
                    alias  Tomas Brod <tomasbrod@azet.sk>
Sent with Free Software. Mutt and Nano.
http://www.gpg4win.org/doc/en/gpg4win-compendium_7.html